tests/system/providers/amazon/aws/example_comprehend_document_classifier.py

```diff
@@ -0,0 +1,234 @@
+# for testing purpose, we will generate 10 copies of each document referenced below.
+PUBLIC_DATA_SOURCES = [
+    {
+        "fileName": "discharge-summary.pdf",
+        "url": "https://github.com/aws-samples/amazon-comprehend-examples/blob/master/building-custom-classifier/sample-docs/discharge-summary.pdf?raw=true",
+    },
+    {
+        "fileName": "doctors-notes.pdf",
+        "url": "https://github.com/aws-samples/amazon-comprehend-examples/blob/master/building-custom-classifier/sample-docs/doctors-notes.pdf?raw=true",
+    },
+]
+
+# Annotations file won't allow headers
+# label,document name,page number
+
+ANNOTATIONS = """DISCHARGE_SUMMARY,discharge-summary-0.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-1.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-2.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-3.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-4.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-5.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-6.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-7.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-8.pdf,1
+DISCHARGE_SUMMARY,discharge-summary-9.pdf,1
+DOCTOR_NOTES,doctors-notes-0.pdf,1
+DOCTOR_NOTES,doctors-notes-1.pdf,1
+DOCTOR_NOTES,doctors-notes-2.pdf,1
+DOCTOR_NOTES,doctors-notes-3.pdf,1
+DOCTOR_NOTES,doctors-notes-4.pdf,1
+DOCTOR_NOTES,doctors-notes-5.pdf,1
+DOCTOR_NOTES,doctors-notes-6.pdf,1
+DOCTOR_NOTES,doctors-notes-7.pdf,1
+DOCTOR_NOTES,doctors-notes-8.pdf,1
+DOCTOR_NOTES,doctors-notes-9.pdf,1"""
+
+
+@task_group
+def document_classifier_workflow():
+    # [START howto_operator_create_document_classifier]
+    create_document_classifier = ComprehendCreateDocumentClassifierOperator(
+        task_id="create_document_classifier",
+        document_classifier_name=classifier_name,
+        input_data_config=input_data_configurations,
+        output_data_config=output_data_configurations,
+        mode="MULTI_CLASS",
+        data_access_role_arn=test_context[ROLE_ARN_KEY],
+        language_code="en",
+        document_classifier_kwargs=document_classifier_kwargs,
+    )
+    # [END howto_operator_create_document_classifier]
+    create_document_classifier.wait_for_completion = False
+
+    # [START howto_sensor_create_document_classifier]
+    await_create_document_classifier = ComprehendCreateDocumentClassifierCompletedSensor(
+        task_id="await_create_document_classifier", document_classifier_arn=create_document_classifier.output
+    )
+    # [END howto_sensor_create_document_classifier]
+
+    @task
+    def delete_classifier(document_classifier_arn: str):
+        ComprehendHook().conn.delete_document_classifier(DocumentClassifierArn=document_classifier_arn)
+
+    chain(
+        create_document_classifier,
+        await_create_document_classifier,
+        delete_classifier(create_document_classifier.output),
+    )
+
+
+@task
+def copy_data_to_s3(bucket: str, sources: list[dict], prefix: str, number_of_copies=1):
```

Comment:
Have you considered using operators such as `HttpToS3Operator` or `LocalFilesystemToS3Operator`?

Reply:
These are actually useful here, somehow i forgot. I have updated now, copy data to use HttpToS3Operator. Thank you @vincbeck. 😄